rtgui/paramsedited.cc

```diff
@@ -3359,6 +3382,46 @@ void ParamsEdited::combine(rtengine::procparams::ProcParams& toEdit, const rteng
         toEdit.sh.lab = mods.sh.lab;
     }
 
+    if (cg.th_c) {
+        toEdit.cg.th_c = mods.cg.th_c;
+    }
```

Comment:
No batch editor add/set behavior?

Reply:
is it useful, because this algorithm is really dedicated to a type of image (Red flowers, Blue LED,...)